### ISSUE TITLE
Improved handling of notification popups

### ISSUE THREAD

**Issue by AllesMeins** [realcopacetic/skin.copacetic#204]
_TL/DR: I'm hoping for an option to hide notifications during movie playback and a progress percentage for notifications_

When using copacetic it seems there is no way to suppress notifications (those infos that pop-up in the top right corner) from showing up while watching a movie.

I'm using kodi with a emby server as a backend. So stuff like adding new media or updating progress can take a bit longer than usual. If there's a lot to process it can take ten or fifteen minutes during which a progress notification is shown. In Copacetic this notification is also visible while watching a movie which is quite distracting. 

![PXL_20241014_191714759 MP](https://github.com/user-attachments/assets/d0e79e87-f462-4232-ace9-784185439a34)
_Copacetic shows a distracting progress notification while playing a movie_

First I thought that I need to blame emby for that, but it turns out that with kodi the skins are supposed to control what is shown during a movie. I noticed this when I temporarily switched back to Estuary which is capable of showing those progress notifications only when no video is playing. If you start a movie the operation continues in the background, but no distracting notifications are displayed. Would this be possible for Copacetic as well?

While I was at it, I also noticed that Estuary also shows a progress percentage, giving you some idea of how long it will take to finish. The same sync progress with Copacetic only shows an increasing number but with no info where it will go to. So I've no idea whether this is just a short update or a major re-sync with probably hundreds of files that need to be processed.

![PXL_20241014_191529204](https://github.com/user-attachments/assets/a21eef1a-0533-4521-99af-e4d9b0f69c10)
_What Copacetic shows_

![PXL_20241014_190957000 MP](https://github.com/user-attachments/assets/79e3ef94-ecab-44e6-88f7-775b1484e2db)
_Same process in Estuary with progress percentage_

I'm using copacetic 1.6.8 with copacetic helper 1.1.10

**Comment by realcopacetic** [comment]
hi I can add a toggle to hide notifications during playback in skin settings. There's also a setting to hide notifications for library updates in Kodi settings you can use in the meantime. Re. progress percentage, I'll look at it - tbh I was planning to redo the notifications anyway as right now, it's too much of a hassle having different style/positioned notifications depending the view you're in so I want to streamline it anyway

**Comment by AllesMeins** [comment]
Thank you, that would be very much appreciated. If I can do anything to help or test just let me know.

**Comment by AllesMeins** [comment]
Due to current events (had to reset my database and now watching my system sync for over half an hour without any idea how much longer I'll have to wait :) ) I thought I revisit this issue. Did you have any chance to take a closer look at this?

**Comment by realcopacetic** [comment]
Sorry I've had a busy couple of months, with work and kids so no free time but will try and get to this soon

**Comment by AllesMeins** [comment]
No worries, I know that you're doing this in your spare time and I really appreciate the work you've done so far. So please don't feel pressured in any way and do it in your own time! Real life always comes first :-)
